docs/language-server/protocol-language-server.md

```diff
@@ -481,8 +481,8 @@ The argument of a [`SuggestionEntry`](#suggestionentry).
 interface SuggestionEntryArgument {
   /** The argument name. */
   name: string;
-  /** The argument type. String 'Any' is used to specify generic types. */
-  reprType: string;
+  /** The argument type. */
+  reprType: string[];
```

Comment:
- issue #13268 talks about
- "what is the type? Is it a list of qualified names of intersection types?"
- I am not sure just chaining `string` to `string[]` really answers the question.
- doesn't TypeScript allow _type aliases_?
- can we define `typename Type string[]` and provide some docs for the type
- and then use `Type` everywhere?
- btw. it might be better to `typename Type (string | string[])` as most of the time values have single FQN